src/test/scala/io/iohk/ethereum/ledger/RewardTellerSpec.scala

```diff
@@ -0,0 +1,44 @@
+package io.iohk.ethereum.ledger
+
+import io.iohk.ethereum.utils.MonetaryPolicyConfig
+import org.scalatest.prop.PropertyChecks
+import org.scalatest.{FlatSpec, Matchers}
+
+class RewardTellerSpec extends FlatSpec with Matchers with PropertyChecks {
+
+  "RewardTeller" should "correctly calculate block and ommer rewards" in {
+    val standardMP = MonetaryPolicyConfig(5000000, 0.2, 5000000000000000000L)
+
+    val testMP = MonetaryPolicyConfig(10, 0.5, 5000000)
+
+    val table = Table[MonetaryPolicyConfig, BigInt, List[BigInt], BigInt, List[BigInt]](
+      ("config", "blockNumber", "ommersNumbers", "expectedBlockReward", "expectedOmmersRewards"),
+      (standardMP, 1, Nil, 5000000000000000000L, Nil),
+      (standardMP, 1000000, List(999999), 5156250000000000000L, List(4375000000000000000L)),
+      (standardMP, 5000000, List(4999998, 4999997), 5312500000000000000L, List(3750000000000000000L, 3125000000000000000L)),
+      (standardMP, 5000001, Nil, 4000000000000000000L, Nil),
```

Comment:
I just had  to add this, to see the changeover. -> `(standardMP, 5000000, Nil, 5000000000000000000L, Nil),`